src/app/components/shared/country-picker/CountryPicker.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Element } from 'nav-frontend-typografi';
+import { Select } from 'nav-frontend-skjema';
+
+import DateInput from './../date-input/DateInput';
+
+import './countryPicker.less';
+
+const CountryPicker = () => (
+    <div>
+        <Select label="">
+            <option value="" />
+            <option value="marshalloyene">Marshalløyene</option>
+        </Select>
+        <div className="dateWrapper">
+            <div>
```

Comment:
Istedenfor å style div-ene her er det litt cleanere om du i .dateWrapper styler på DateInput tenker jeg, noe ala:

`
.dateWrapper {
    display: flex
    .dateInput:first-child {
         margin-right: @marginMedium;
    }
`

Reply:
Så kan de to ekstra div-ene fjernes fra markupen :)

Reply:
Div-ene kunne fjernes, men .dateInput er ingenting

Reply:
Klassen dateInput blir satt i DateInput-komponenten. Bruk av den prop-en vil sørge for at det settes på input-feltet. Vet ikke om det er intentional? Hvis du vil ha det på div-en for skjemaelementet input, kan vanlig className-prop på Input-komponenten benyttes til det.